clang/test/CAS/fdepscan-result-cache-output-path.c

```diff
@@ -0,0 +1,15 @@
+// TODO: Test should be updated to not depend on a working cache at default location, which is out of test/build directory.
+//
+/// Test absolute path output.
+// RUN: rm -rf %t && mkdir %t
+// RUN: %clang -target x86_64-apple-macos11 -fdepscan=inline -c -x c %s -o %t/out.o
+// RUN: ls %t/out.o && rm %t/out.o
+// RUN: %clang -target x86_64-apple-macos11 -fdepscan=inline -c -x c %s -o %t/out.o
```

Comment:
It's not clear why this is being run twice, but it would be good to check whether there's a cache hit/miss with `-Rresult-cache`. Relatedly, I think @akyrtzi landed support for `-fcas-builtin-path` with `-fdepscan`. That means you can use a fresh CAS in `%t` and get a known miss on the first result.